diff --git a/src/module/repo/ostree_repohelper.cpp b/src/module/repo/ostree_repohelper.cpp
index ecaa9c6..03af650 100644
--- a/src/module/repo/ostree_repohelper.cpp
+++ b/src/module/repo/ostree_repohelper.cpp
@@ -504,8 +504,7 @@ bool OstreeRepoHelper::startOstreeJob(const QString &cmd, const QString &ref, co
     qint64 processId = process.processId();
     // 通过script pid 查找对应的ostree pid
     if ("script" == cmd) {
-        qint64 shPid = getChildPid(processId);
-        qint64 ostreePid = getChildPid(shPid);
+        qint64 ostreePid = getChildPid(processId);
         jobMap.insert(ref, ostreePid);
     }
     if (!process.waitForFinished(timeout)) {
